<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Merkaartor</string>
  </property>
  <widget class="QWidget" name="centralWidget" />
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="helpAboutAction" />
   </widget>
   <widget class="QMenu" name="menuCreate" >
    <property name="title" >
     <string>&amp;Create</string>
    </property>
    <addaction name="createNodeAction" />
    <addaction name="createRoadAction" />
    <addaction name="createDoubleWayAction" />
    <addaction name="createRoundaboutAction" />
    <addaction name="createAreaAction" />
    <addaction name="createRelationAction" />
   </widget>
   <widget class="QMenu" name="menuRoad" >
    <property name="title" >
     <string>&amp;Road</string>
    </property>
    <addaction name="roadSplitAction" />
    <addaction name="roadBreakAction" />
    <addaction name="roadJoinAction" />
    <addaction name="editReverseAction" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="editUndoAction" />
    <addaction name="editRedoAction" />
    <addaction name="editRemoveAction" />
    <addaction name="separator" />
    <addaction name="editSelectAction" />
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuBookmarks" >
     <property name="title" >
      <string>&amp;Bookmarks</string>
     </property>
     <addaction name="bookmarkAddAction" />
     <addaction name="bookmarkRemoveAction" />
     <addaction name="separator" />
    </widget>
    <widget class="QMenu" name="mnuProjections" >
     <property name="title" >
      <string>Set &amp;projection</string>
     </property>
     <addaction name="separator" />
    </widget>
    <addaction name="viewZoomAllAction" />
    <addaction name="viewZoomWindowAction" />
    <addaction name="viewZoomOutAction" />
    <addaction name="viewZoomInAction" />
    <addaction name="separator" />
    <addaction name="viewSetCoordinatesAction" />
    <addaction name="menuBookmarks" />
    <addaction name="separator" />
    <addaction name="mnuProjections" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuExport_OSM" >
     <property name="title" >
      <string>&amp;Export OSM</string>
     </property>
     <addaction name="exportOSMAllAction" />
     <addaction name="exportOSMViewportAction" />
    </widget>
    <addaction name="fileNewAction" />
    <addaction name="fileOpenAction" />
    <addaction name="fileImportAction" />
    <addaction name="menuExport_OSM" />
    <addaction name="renderAction" />
    <addaction name="separator" />
    <addaction name="fileDownloadAction" />
    <addaction name="fileUploadAction" />
    <addaction name="separator" />
    <addaction name="fileQuitAction" />
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>&amp;Tools</string>
    </property>
    <widget class="QMenu" name="menuStyle_2" >
     <property name="title" >
      <string>&amp;Style</string>
     </property>
     <addaction name="editMapStyleAction" />
     <addaction name="separator" />
     <addaction name="mapStyleSaveAction" />
     <addaction name="mapStyleLoadAction" />
    </widget>
    <addaction name="menuStyle_2" />
    <addaction name="separator" />
    <addaction name="toolsPreferencesAction" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuView" />
   <addaction name="menuCreate" />
   <addaction name="menuRoad" />
   <addaction name="menuTools" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="StatusBar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="windowTitle" >
    <string>Tool Bar</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="editUndoAction" />
   <addaction name="editRedoAction" />
   <addaction name="separator" />
   <addaction name="editPropertiesAction" />
   <addaction name="editRemoveAction" />
   <addaction name="editMoveAction" />
   <addaction name="editAddAction" />
   <addaction name="editReverseAction" />
  </widget>
  <action name="fileQuitAction" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="helpAboutAction" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="fileOpenAction" >
   <property name="text" >
    <string>&amp;Open</string>
   </property>
   <property name="statusTip" >
    <string>Create a new document and import a file</string>
   </property>
  </action>
  <action name="viewZoomAllAction" >
   <property name="text" >
    <string>Zoom &amp;all</string>
   </property>
   <property name="shortcut" >
    <string>F2</string>
   </property>
  </action>
  <action name="viewZoomWindowAction" >
   <property name="text" >
    <string>Zoom &amp;window</string>
   </property>
   <property name="iconText" >
    <string>Zoom window</string>
   </property>
   <property name="toolTip" >
    <string>Zoom window</string>
   </property>
   <property name="shortcut" >
    <string>F3</string>
   </property>
  </action>
  <action name="viewZoomOutAction" >
   <property name="text" >
    <string>Zoom &amp;out</string>
   </property>
  </action>
  <action name="viewZoomInAction" >
   <property name="text" >
    <string>Zoom &amp;in</string>
   </property>
   <property name="iconText" >
    <string>Zoom in</string>
   </property>
   <property name="toolTip" >
    <string>Zoom in</string>
   </property>
  </action>
  <action name="createWayAction" >
   <property name="text" >
    <string>Curved link</string>
   </property>
   <property name="iconText" >
    <string>Curved link</string>
   </property>
   <property name="toolTip" >
    <string>Curved link</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="editUndoAction" >
   <property name="icon" >
    <iconset resource="Icons/AllIcons.qrc" >:/Icons/edit-undo.xpm</iconset>
   </property>
   <property name="text" >
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="editRedoAction" >
   <property name="icon" >
    <iconset resource="Icons/AllIcons.qrc" >:/Icons/edit-redo.xpm</iconset>
   </property>
   <property name="text" >
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="editMoveAction" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="Icons/AllIcons.qrc" >:/Icons/move.xpm</iconset>
   </property>
   <property name="text" >
    <string>Move</string>
   </property>
   <property name="shortcut" >
    <string>M</string>
   </property>
  </action>
  <action name="fileImportAction" >
   <property name="text" >
    <string>&amp;Import</string>
   </property>
   <property name="statusTip" >
    <string>Import a file into the current document</string>
   </property>
  </action>
  <action name="fileDownloadAction" >
   <property name="text" >
    <string>&amp;Download from web</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="createLinearWayAction" >
   <property name="text" >
    <string>Link</string>
   </property>
   <property name="iconText" >
    <string>Create link</string>
   </property>
   <property name="toolTip" >
    <string>Create link</string>
   </property>
   <property name="shortcut" >
    <string>L</string>
   </property>
  </action>
  <action name="editPropertiesAction" >
   <property name="icon" >
    <iconset resource="Icons/AllIcons.qrc" >:/Icons/toolselect.xpm</iconset>
   </property>
   <property name="text" >
    <string>Edit</string>
   </property>
   <property name="shortcut" >
    <string>Esc</string>
   </property>
  </action>
  <action name="fileUploadAction" >
   <property name="text" >
    <string>&amp;Upload to web</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="editRemoveAction" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="Icons/AllIcons.qrc" >:/Icons/remove.xpm</iconset>
   </property>
   <property name="text" >
    <string>R&amp;emove</string>
   </property>
   <property name="shortcut" >
    <string>Del</string>
   </property>
  </action>
  <action name="createRoadAction" >
   <property name="text" >
    <string>&amp;Road</string>
   </property>
   <property name="shortcut" >
    <string>R</string>
   </property>
  </action>
  <action name="editAddAction" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="Icons/AllIcons.qrc" >:/Icons/add.xpm</iconset>
   </property>
   <property name="text" >
    <string>Add</string>
   </property>
  </action>
  <action name="createNodeAction" >
   <property name="text" >
    <string>&amp;Node</string>
   </property>
   <property name="shortcut" >
    <string>N</string>
   </property>
  </action>
  <action name="editReverseAction" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="Icons/AllIcons.qrc" >:/Icons/reverse.xpm</iconset>
   </property>
   <property name="text" >
    <string>&amp;Reverse</string>
   </property>
  </action>
  <action name="viewSetCoordinatesAction" >
   <property name="text" >
    <string>&amp;Set coordinates</string>
   </property>
  </action>
  <action name="createDoubleWayAction" >
   <property name="text" >
    <string>&amp;Double carriage way</string>
   </property>
  </action>
  <action name="createRoundaboutAction" >
   <property name="text" >
    <string>&amp;Roundabout</string>
   </property>
  </action>
  <action name="fileNewAction" >
   <property name="text" >
    <string>&amp;New</string>
   </property>
   <property name="statusTip" >
    <string>Create a new document</string>
   </property>
  </action>
  <action name="roadSplitAction" >
   <property name="text" >
    <string>&amp;Split</string>
   </property>
  </action>
  <action name="roadJoinAction" >
   <property name="text" >
    <string>&amp;Join</string>
   </property>
  </action>
  <action name="roadBreakAction" >
   <property name="text" >
    <string>&amp;Break apart</string>
   </property>
  </action>
  <action name="createRelationAction" >
   <property name="text" >
    <string>Re&amp;lation</string>
   </property>
  </action>
  <action name="createAreaAction" >
   <property name="text" >
    <string>&amp;Area</string>
   </property>
  </action>
  <action name="editMapStyleAction" >
   <property name="text" >
    <string>&amp;Edit</string>
   </property>
  </action>
  <action name="mapStyleSaveAction" >
   <property name="text" >
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="mapStyleLoadAction" >
   <property name="text" >
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="createCurvedRoadAction" >
   <property name="text" >
    <string>&amp;Curved road</string>
   </property>
  </action>
  <action name="toolsPreferencesAction" >
   <property name="text" >
    <string>&amp;Preferences...</string>
   </property>
  </action>
  <action name="exportOSMAllAction" >
   <property name="text" >
    <string>&amp;All...</string>
   </property>
   <property name="statusTip" >
    <string>Export all visible layers to a file</string>
   </property>
  </action>
  <action name="editSelectAction" >
   <property name="text" >
    <string>&amp;Select...</string>
   </property>
  </action>
  <action name="renderAction" >
   <property name="text" >
    <string>Render...</string>
   </property>
  </action>
  <action name="exportOSMViewportAction" >
   <property name="text" >
    <string>&amp;Viewport...</string>
   </property>
   <property name="statusTip" >
    <string>Export the features in the viewport to a file</string>
   </property>
  </action>
  <action name="bookmarkAddAction" >
   <property name="text" >
    <string>&amp;Add...</string>
   </property>
  </action>
  <action name="bookmarkRemoveAction" >
   <property name="text" >
    <string>&amp;Remove...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Icons/AllIcons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>fileQuitAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>403</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
